README.md

```diff
@@ -118,100 +114,35 @@
-
-    <httpProtocol>
-        <customHeaders>
-            <add name="Access-Control-Allow-Headers" value="Content-Type, Accept, X-Requested-With,  Authorization" />
-            <add name="Access-Control-Allow-Methods" value="GET, POST, PUT, DELETE, OPTIONS" />
-            <add name="Access-Control-Allow-Origin" value="*" />
-        </customHeaders>
-    </httpProtocol>
-
-## 1.5 Http Status Codes
-
-The BCF API relies on the regular Http Status Code definitions. Good sources are [Wikipedia](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes) or the [HTTP/1.1 Specification](https://tools.ietf.org/html/rfc7231).
+## 1.1 Open CDE Foundation API
 
-Generally, these response codes shall be used in the API:
-* `200 - OK` for `GET` requests that return data or `PUT` requests that update data
-* `201 - Created` for `POST` requests that create data
+BCF API is a member of the Open CDE API family. All Open CDE APIs are united by a shared common API called [Open CDE Foundation API](https://github.com/buildingSMART/shared-common-API). 
+The foundation API specifies a small number of services and a few conventions that are common to all Open CDE APIs. All BCF API implementations must implement the Foundation API and follow its conventions and guidelines. 
+Implementers should start by implementing the Foundation API and only then continue to implement the BCF API.
 
-`POST` and `PUT` requests do usually include the created resource in the response body. Exceptions to this rule are described in the specific section for the resource.
-
-## 1.6 Error Response Body Format
-
-BCF-API has a specified error response body format [error.json](Schemas_draft-03/error.json).
-
-## 1.7 DateTime Format
-
-DateTime values in this API are supposed to be in ISO 8601 compliant `YYYY-MM-DDThh:mm:ss` format with optional time zone indicators. This is the same format as defined in the Xml `xs:dateTime` type as well as the result of JavaScripts Date.toJson() output.
-
-For example, `2016-04-28T16:31:12.270+02:00` would represent _Thursday, April 28th, 2016, 16:31:12 (270ms) with a time zone offset of +2 hours relative to UTC._
-Please note that the colon in the timezone offset is optional, so `+02:00` is equivalent to `+0200`.
-
-To void ambiguity, This specification steps away from ISO 8601 on the topic of DateTime values with no timezone: The ISO 8601 says that DateTime values with no timezone designator are local times - **In BCF all DateTime values with no timezone designator as assumed to be in UTC**.
-
-## 1.8 Authorization
+## 1.2 Authorization
 
 API implementors can optionally choose to restrict the actions a user is allowed to perform on the BCF entities
 via the API. The global default authorizations for all entities are expressed in the project extensions schema and can
 be locally overridden in the entities themselves.
 
-### 1.8.1 Per-Entity Authorization
+### 1.2.1 Per-Entity Authorization
 
```

Comment:
There was a proposal to always include authorizations, so this might be obsolete

Reply:
@pbuts that proposal was rejected